bertytech/content/blog/bluetooth-low-energy/index.md

```diff
@@ -0,0 +1,222 @@
+---
+author: aeddi
+title: Berty, Libp2p and Bluetooth Low Energy
+date: 2020-04-08
+image: "https://i.ibb.co/BNBbZxH/2020-04-06-Berty-blogpost-Bluetooth-Low-Energy.jpg"
+categories:
+- announcement
+tags:
+- technical
```

Comment:
announcement aussi je dirai

Reply:
I updated the tags, but I don't share your feeling about "announcement" for this article, maybe other people can help us choose